pyquil/paulis.py

```diff
@@ -468,7 +468,7 @@ def __pow__(self, power):
         """
         if not isinstance(power, int) or power < 0:
             raise ValueError("The power must be a non-negative integer.")
-        result = 1
+        result = ID
```

Comment:
Should this be `PauliTerm(ID)` to avoid `ID` from being mutated.
I think `ID` should have been a function